broker/endpoints/Patient.js

```diff
@@ -276,6 +277,9 @@ async function createPatient(patient, head) {
   const createdPatient = await axios.post(`/Patient`, resource);
   patientID.resourceId = createdPatient.data.id;
 
+  // Create the new Patient/Client record in TimeTap
+  await tt_service.createTimetapClient(createdPatient.data);
```

Comment:
Should we be wrapping all this in a try/catch or a Transaction type logic so that if either our FHIR write or the TimeTap write fails, then we can throw an error back to front end without creating random orphaned records?